components/resources/ai_chat_prompts.grdp

```diff
@@ -151,4 +161,22 @@ Your goal is to answer the user's requests exactly in concise manner.
 Sure, here are three intriguing or unusual questions that a reader may ask about the page: &lt;ul&gt; &lt;li&gt;
 </message>
 
+<message name="IDS_AI_CHAT_LLAMA2_GENERATE_REWRITE_SUGGESTION_PROMPT" translateable="false">
+This is an excerpt user selected to be rewritten:
+&lt;excerpt&gt;
+<ph name="EXCERPT">$1</ph>
+&lt;/excerpt&gt;
+
+<ph name="USER_REQUEST">$2</ph>
+</message>
+
+<message name="IDS_AI_CHAT_LLAMA2_SYSTEM_MESSAGE_GENERATE_REWRITE_SUGGESTION" translateable="false">
+Your name is Leo, a helpful, respectful and honest AI assistant created by the company Brave. You will be replying to a user of the Brave browser. Always respond in a neutral tone. Be polite and courteous. Answer concisely in no more than 50-80 words. Don't append word counts at the end of your replies.
+Your goal is to help user rewrite the excerpt and only include the rewritten texts so user can copy and paste your response without any modification.
```

Comment:
One of the reason that I use this rather than a simpler system prompt was because we have some hardcoded prompt check on ai_chat server end, not sure if we want to add a new one there instead and make it simpler here. So far this seems working fine tho.